cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,50 @@
 /// <reference types='cypress' />
+import faker from 'faker';
+import HomeAndCataloguePageObject
+  from '../support/pages/homeСatalogue.pageObject';
 
-describe('', () => {
-  before(() => {
+const homePage = new HomeAndCataloguePageObject();
+const testData = {
+  name: faker.name.firstName(),
+  country: faker.random.words(),
+  city: faker.random.words(),
+  card: faker.finance.creditCardNumber(),
+  year: 2023,
+  month: 12
+};
 
+describe('Product store', () => {
+  beforeEach(() => {
+    cy.visit('/');
   });
 
-  it('', () => {
+  // eslint-disable-next-line max-len
+  it('should have an ability to add product to the card and make checkout', () => {
+    homePage.clickOnCategory('Laptops');
+    homePage.clickOnProduct('Sony vaio i7');
+    homePage.addToCart('Add to cart');
 
+    cy.on('window:alert', (alert) => {
+      expect(alert).to.equal(`Product added`);
+    });
```

Comment:
```suggestion
    cy.on('window:alert', (alert) => {
      expect(alert).to.equal(`Product added`);
    });
```
move this to page object

Reply:
https://github.com/mate-academy/qa_cypress_e2e_demoblaze_page_object/pull/181/commits/59aeb749fc99072325a53083c2bbed19eb0519e8